bookmarklet.js

```diff
@@ -139,19 +138,17 @@ async function convert(code, options) {
       let { path, opts } = extractOptions(s);
       code = loadScript(code, path, opts.loadOnce);
     });
-    code = await minify(code);
   }
 
   if (options.style) {
     options.style.forEach(s => {
       let { path, opts } = extractOptions(s);
       stylesCode = loadStyle(stylesCode, path, opts.loadOnce);
     });
-    const minifiedStyles = await minify(stylesCode);
-    code = minifiedStyles + code;
+    code = stylesCode + code;
   }
 
-  code = `(function(){${code}})()`;
+  code = await minify(`(function(){${code}})()`);
```

Comment:
Minify the whole IIFE produces unexpected result.

You can test the generated output by

```bash
./bin/cli.js test/bookmarklets/test1.bookmarklet.js -d > o1.html
```

then open the file and click the `Test` button within.

In this commit the result is a `true` appear, while the expected result should be an alert.